github/apps.go

```diff
@@ -183,6 +189,29 @@ func (s *AppsService) CreateInstallationToken(ctx context.Context, id int64) (*I
 	return t, resp, nil
 }
 
+// Create a new attachment on user comment containing a url
+//
+// Github API docs: https://developer.github.com/v3/apps/#create-a-content-attachment
```

Comment:
s/Github/GitHub/

Reply:
Hi @gmlewis, can you please elaborate the required change ?

Reply:
Yes, please change "Github" to "GitHub".  Thanks.

Reply:
My bad